README.md

```diff
@@ -1,28 +1,34 @@
-* Builds can be configured to fail based on the results of the automated accessibility tests, preventing both new accessibility bugs and regressions
-* Failure details and _how to fix_ information can be viewed in Azure DevOps under Builds, making it quick and easy to investigate and resolve the accessibility issues
 
-This repository contains sample projects (see next section [Available samples](#available-samples)) demonstrating how to implement automated accessibility testing in [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) builds using [axe-core](https://github.com/dequelabs/axe-core), the same accessibility scanning engine used in [Accessibility Insights for Web](https://accessibilityinsights.io/docs/en/web/overview).
+- Automated accessibility tests run during Continuous Integration and Pull Request builds, ensuring that all code changes are free of common easily-detected accessibility issues before they go to production
+- Builds can be configured to fail based on the results of the automated accessibility tests, preventing both new accessibility bugs and regressions
+- Failure details and _how to fix_ information can be viewed in Azure DevOps under Builds, making it quick and easy to investigate and resolve the accessibility issues
+
+This repository contains sample projects (see next section [Available samples](#available-samples)) demonstrating how to implement automated accessibility testing in [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) builds using [axe-core](https://github.com/dequelabs/axe-core), the same accessibility scanning engine used in [Accessibility Insights for Web](https://accessibilityinsights.io/docs/en/web/overview), and [Axe.Windows](https://github.com/microsoft/axe-windows), the same accessibility scanning engine used in [Accessibility Insights for Windows](https://accessibilityinsights.io/docs/en/windows/overview/).
 
 ## Available samples
 
 The following sample projects specify the main technologies used. A team that uses comparable tools and frameworks should be able to refer to the sample and update their existing tests to incorporate automated accessibility checks.
 
-* **[Sample 1: typescript-selenium-webdriver-sample](./typescript-selenium-webdriver-sample)**: 
-  * Useful for teams using Selenium and TypeScript/JavaScript. 
-  
-* **[Sample 2: CSharpSeleniumWebdriverSample](./csharp-selenium-webdriver-sample)**: 
-  * Useful for teams using Selenium and C#. 
+- **[Sample 1: typescript-selenium-webdriver-sample](./typescript-selenium-webdriver-sample)**:
+
+  - Useful for teams using Selenium and TypeScript/JavaScript.
+
+- **[Sample 2: CSharpSeleniumWebdriverSample](./csharp-selenium-webdriver-sample)**:
+
+  - Useful for teams using Selenium and C#.
+
+- **[Sample 3: WindowsAppSamples](./windows-app-samples)**:

```

Comment:
I like the idea of pointing to the axe.windows docs as an incremental step towards including sample content here, but I think it's a little misleading for us to list it as a "sample" unless we actually have a working example, not just a little documentation. Maybe a new section called something like "Other resources" or "Other documentation" or something?

Reply:
Done